src/chrome/content/rules/viu.tv.xml

```diff
@@ -0,0 +1,13 @@
+<ruleset name="ViuTV">
+	<target host="viu.tv" />
+	<target host="www.viu.tv" />
+	<target host="edm.viu.tv" />
+	<target host="event.viu.tv" />
+	<target host="static.viu.tv" />
+
+	<exclusion pattern="^http://(www\.)?viu.tv/encore/" />
```

Comment:
You need to escape all the dots.
Please also add a comment about the reason for the exclusion.